src/bin/cql-stress-scylla-bench/main.rs

```diff
@@ -146,7 +146,7 @@ async fn prepare(args: Arc<ScyllaBenchArgs>, stats: Arc<ShardedStats>) -> Result
         rate_limit_per_second,
         operation_factory,
         max_retries_per_op: args.max_retries_per_op as usize,
-        max_retries_in_total: usize::MAX,
+        max_retries_in_total: args.max_retries_in_total as usize,
```

Comment:
Hmm, I used `usize` here because `max_retries_per_op` was also of this type. I'll see if changing the type of `max_retries_per_op` to `u64` makes sense.

Reply:
OK, done. There are indeed less casts now.